tests/Unit/ViewComposers/CountryLocaleViewComposerTest.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-namespace PodPoint\I18n\Tests\ViewComposers;
+namespace PodPoint\I18n\Tests\Unit\ViewComposers;
 
-use PHPUnit\Framework\TestCase;
-use PodPoint\I18n\ViewComposers\CountryLocaleViewComposer;
 use Illuminate\View\View;
 use Illuminate\Config\Repository;
+use PodPoint\I18n\Tests\TestCase;
+use PodPoint\I18n\ViewComposers\CountryLocaleViewComposer;
 
-class TestCountryLocaleViewComposer extends TestCase
+class CountryLocaleViewComposerTest extends TestCase
```

Comment:
Maybe it would be good to add IE locale to the tests so we can demonstrate that the issue is fixed?

Reply:
Not sure... I don't really like this test anyway... Will refactor it.